locales-en-US.xml

```diff
@@ -249,6 +369,58 @@
     </term>
 
     <!-- LONG ROLE FORMS -->
+    <term name="chair">
+      <single>chair</single>
+      <multiple>chairs</multiple>
+    </term>
+    <term name="compiler">
+      <single>compiler</single>
+      <multiple>compilers</multiple>
+    </term>
+    <term name="contributor">
+      <single>contributor</single>
+      <multiple>contributors</multiple>
+    </term>
+    <term name="curator">
+      <single>curator</single>
+      <multiple>curators</multiple>
+    </term>
+    <term name="executive-producer">
+      <single>executive producer</single>
+      <multiple>executive producers</multiple>
+    </term>
+    <term name="guest">
+      <single>guest</single>
+      <multiple>guests</multiple>
+    </term>
+    <term name="host">
+      <single>host</single>
+      <multiple>hosts</multiple>
+    </term>
+    <term name="narrator">
+      <single>narrator</single>
+      <multiple>narrators</multiple>
+    </term>
+    <term name="organizer">
+      <single>organizer</single>
+      <multiple>organizers</multiple>
+    </term>
+    <term name="performer">
+      <single>performer</single>
+      <multiple>performers</multiple>
+    </term>
+    <term name="producer">
+      <single>producer</single>
+      <multiple>producers</multiple>
+    </term>
+    <term name="script-writer"> <!-- shouldn't that be scriptwriter?-->
+      <single>scriptwriter</single>
+      <multiple>scriptwriters</multiple>
```

Comment:
```suggestion
      <single>writer</single>
      <multiple>writers</multiple>
```

> `<!-- shouldn't that be scriptwriter?-->`

It was originally just "writer". The "script" was added during review to reduce confusion. I don't have a problem either way, but the variable name that's incorporated into pandoc, etc. at this point is `script-writer`, so we should probably keep it as is.

@adam3smith

Reply:
Pandoc etc can still be changed easily so that's not that important.